src/py42/sdk/queries/fileevents/file_event_query.py

```diff
@@ -53,6 +59,7 @@ def __iter__(self):
             u"pgSize": self.page_size,
             u"srtDir": self.sort_direction,
             u"srtKey": self.sort_key,
+            u"pgToken": self.pgToken,
```

Comment:
This is the reason you are getting 500. It is supposed to be `pageToken`, not `pgToken`